### DISCUSSION TITLE
Customize menu items in OsmAnd

### DISCUSSION THREAD

**Discussion by thilokab** [osmandapp/OsmAnd#11088] [Q&A]
We would like to know if there is a way to customize the current OSMAnd menu that is displayed when the users swipe from the left. We are using OsmAndHelper.java to utilize navigation through our mobile application.

We would like to hide most of the menu items that are currently available as shown in the attached image.

![Screenshot_20210305-101723_OsmAnd](https://user-images.githubusercontent.com/78388857/110044531-68694900-7d9d-11eb-8022-c7b30a22efb8.jpg)

**Comment by vshcherb** [comment]
Customization of menu is possible via UI -> Settings -> Customize Appearance -> Main menu. Then you could export profile and get 'package.osf'. API provides various ways to import "package.osf" also there are many other things configurable via UI with profile settings which are not present in AIDL API method.
